src/spotted/handlers/job_handlers.py

```diff
@@ -75,3 +75,20 @@ async def db_backup_job(context: CallbackContext):
             )
         except BinasciiError as ex:
             await context.bot.send_message(chat_id=admin_group_id, text=f"✖️ Impossibile effettuare il backup\n\n{ex}")
+
+
+async def clean_muted_users(context: CallbackContext):
+    """Job called each day at 05:00 utc.
+    Removed expired users mute records from the database
+
+    Args:
+        context: context passed by the jobqueue
+    """
+    expired_muted = DbManager.select_from(
+        table_name="muted_users", select="user_id", where="expire_date < DATETIME('now')"
+    )
+    if len(expired_muted) == 0:
+        return
+    for user in expired_muted:
```

Comment:
you could put a newline before

Reply:
Newlines there will make pylint cry. It will decrease by 0.01 the code quality score.